odd-platform-ui/src/components/DataEntityDetails/Overview/OverviewMetadata/MetadataItem/MetadataItem.tsx

```diff
@@ -17,16 +17,9 @@ import EditIcon from 'components/shared/Icons/EditIcon';
 import AppButton from 'components/shared/AppButton/AppButton';
 import AppIconButton from 'components/shared/AppIconButton/AppIconButton';
 import MetadataValueEditor from 'components/DataEntityDetails/Metadata/MetadataValueEditor/MetadataValueEditor';
-import {
-  Container,
-  LabelContainer,
-  Actions,
-  EditForm,
-  Label,
-  Value,
-  ValueContainer,
-  FormActionBtns,
-} from './MetadataItemStyles';
+import * as S from './MetadataItemStyles';
+import CopyButton from '../../../../shared/CopyButton/CopyButton';
+import DropdownIcon from '../../../../shared/Icons/DropdownIcon';
```

Comment:
Please, use relative paths for closest components imports, here will be better to use absolute path.

Reply:
done